modules/ob-installing-builds-with-operator.adoc

```diff
@@ -28,7 +28,7 @@ You can install the {builds-operator} from the OperatorHub by using the {builds-
 .. Select an *Update Channel*:
 
 ** The *Update Channel* is set to *latest* by default. The latest channel enables installation of the most recent stable version of the {builds-operator}.
-** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.1`, you can use the `builds-1.1` channel.
+** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.2`, you can use the `builds-1.2` channel and so on.
```

Comment:
```suggestion
** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.2`, you can use the `builds-1.2` channel. 
```

"and so on" is usually used to end the list. Since it's just one example, "and so on" is not needed here.